boostregex/UTF8DocumentIterator.cxx

```diff
@@ -1,10 +1,14 @@
 ﻿#include "UTF8DocumentIterator.h"
 
 
+#include <string>

```

Comment:
On my local build, including `<string>` isn't required, only `<string_view>` and `<stdexcept>`.

Reply:
Yes, I just added it because of some copy&paste from BoostRegExSearch.cxx and <string> is not needed here.